CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix caching of re-encoded video files (#82)
 - Do not start multiple video processing threads by default (`--processes` default value) (second part of #83)
 - Fix logging issue in DEBUG mode
+- Refactored some shell scripts in `CONTRIBUTING.md` (#91)
```

Comment:
This has to be added to the `[Unreleased]` section, not to the already released 1.2.0

I would rephrase this to "Enhance contribution guidelines", which is a bit too vague, but more inline with your intent.

Reply:
Done